constellations/gemini-user/src/achievements/achievements.sass

```diff
@@ -28,7 +28,8 @@
             background-color: #b8b8b8
             
         & p
-            margin: 0 0 0 32px
+            margin: 16px 0 16px 32px
+            overflow-wrap: anywhere
```

Comment:
I think `overflow-wrap` is not supported by Internet Explorer 11 yet. You may also have to specify `word-wrap` (https://www.w3schools.com/cssref/css3_pr_word-wrap.asp)